element_deeplabcut/model.py

```diff
@@ -300,9 +300,11 @@ def insert_new_model(cls, model_name: str, dlc_config: dict, *, shuffle: int,
             return
         with cls.connection.transaction:
             cls.insert1(model_dict)
-            # 'not None' bc, when should execute, returns list w/ambiguous truth val
-            if BodyPart.extract_new_body_parts(dlc_config, verbose=False) is not None:
+            # Returns array, so check size for unambiguous truth value
+            if BodyPart.extract_new_body_parts(dlc_config, verbose=False).size > 0:

```

Comment:
```suggestion
            if len(BodyPart.extract_new_body_parts(dlc_config, verbose=False)):

```

Reply:
Thanks for your review @ttngu207. Resolved [both suggestions with this commit](https://github.com/CBroz1/element-deeplabcut/pull/7/commits/85f9f8d2dd4fac1c2e5cbf3f9d0475cb5729fe64). Ready to merge.

Is there a reason to prefer `len(array)` to `array.size > 0`? Perhaps the `size` method takes longer/more memory